exercises/custom-converter/practice/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "durable-execution",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc --build",
+    "build.watch": "tsc --build --watch",
+    "lint": "eslint .",
+    "start": "ts-node src/worker.ts",
+    "start.watch": "nodemon src/worker.ts",
+    "workflow": "ts-node src/client.ts",
+    "service": "ts-node src/service.ts",
+    "service.watch": "nodemon src/service.ts",
+    "format": "prettier --config .prettierrc 'src/**/*.ts' --write"
+  },
+  "nodemonConfig": {
+    "execMap": {
+      "ts": "ts-node"
+    },
+    "ext": "ts",
+    "watch": [
+      "src"
+    ]
+  },
+  "dependencies": {
+    "@temporalio/activity": "~1.8.0",
```

Comment:
Any reason not to use ^1.9.0 for all `@temporalio/*` deps?

Reply:
Nope, we've just been bumping deps gradually across our courses (we only just bumped ts 101 to ^1.9.0 a couple weeks ago) and these often end up getting copy and pasted, thanks for catching.

Reply:
Thanks, have done this and made a ticket for myself to do this for all courses: https://temporalio.atlassian.net/browse/EDU-2282